src/OrchardCore.Themes/TheBlogTheme/Views/Layout.liquid

```diff
@@ -48,7 +48,7 @@
 
             <div id="bs-example-navbar-collapse-1" class="collapse navbar-collapse">
 
-                {% helper "shape", type: "menu", alias: "alias:main-menu", cache_id: "main-menu", cache_duration: "00:05:00", cache_tag: "alias:main-menu" %}
+                {% shape type: "menu", alias: "alias:main-menu", cache_id: "main-menu", cache_duration: "00:05:00", cache_tag: "alias:main-menu" %}
```

Comment:
Could `type` be the default argument, and then we could type `{% shape "menu", ... %}